packages/@apphosting/publish-dev/verdaccio-config.yaml

```diff
@@ -0,0 +1,76 @@
+#
+# This is the default configuration file. It allows all users to do anything,
+# please read carefully the documentation and best practices to
+# improve security.
```

Comment:
We should probably remove this bit about the default configuration file since it looks like you've made some modifications